backend/stewards/tests.py

```
@@ -38,3 +40,159 @@ def test_regular_user_cannot_mutate_arbitrary_steward_profile(self):
+        response = self.client.post(
+            f'/api/v1/stewards/feature-reviews/{self.submission.id}/score/',
+            {'score': 2},
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+        self.assertFalse(FeatureCandidateScore.objects.exists())
+
+    def test_staff_admin_sees_aggregates_and_manual_review_flag(self):
+        second_user = User.objects.create_user(
+            email='second-reviewer@example.com',
+            password='testpass123',
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to argument: "password"

(S106)